workspaces/optic-engine-wasm/src/lib.rs

```diff
@@ -380,6 +380,28 @@ pub fn spec_resolve_response(
     .map_err(|err| JsValue::from(format!("responses could not be serialized: {:?}", err)))
 }
 
+#[wasm_bindgen]
+pub fn spec_endpoint_delete_commands(
+  spec: &WasmSpecProjection,
+  path_id: String,
+  method: String,
+) -> Result<String, JsValue> {
+  let endpoint_queries = spec.endpoint_queries();
+
+  let commands = endpoint_queries
+    .delete_endpoint_commands(&path_id, &method)
+    .ok_or_else(|| {
+      JsValue::from("delete endpoint commands could not be generated for unexisting endpoint")
+    })?;
```

Comment:
Since we accept arbitrary strings for both the `path_id` and `method`, we have to deal with the case where commands are requested for an endpoint that doesn't exist. Here we map the `Option<T>` to a `Result<T, JsValue>`, which is so far the best way we've found to forward errors.